### ISSUE TITLE
Featured bundle keeps resetting after removal

### ISSUE THREAD

**Issue by frnsys** [misfist/tni-core-functionality#46]
hey @misfist, we're trying to remove the featured post bundle. If we unset it in the Featured Content backend, it goes away, but for whatever reason, returns after some time. Any idea why this could be happening?

**Comment by misfist** [comment]
@frnsys - Sorry, this fell through the cracks. 

> If we unset it in the Featured Content backend, it goes away, but for whatever reason, returns after some time. Any idea why this could be happening?

I don't know why this would be.

When, I delete the bundle from the Featured Content option page, it's value becomes `''` in the database. So, the condition to check for a bundle fails.

https://github.com/thenewinquiry/tni-core-functionality/blob/master/includes/helpers.php#L159-L195

Testing locally showed this to be working. However, I did notice that the option selector _appeared_ to have a bundle value after clearing it. The DB had an empty value though.

I made a change to the field so that it doesn't appear to have a value after removing it.
